iroh/src/client/blobs.rs

```diff
@@ -55,6 +65,18 @@ impl<C> Client<C>
 where
     C: ServiceConnection<RpcService>,
 {
+    /// Create a new batch for adding data.
+    pub async fn batch(&self) -> Result<Batch<C>> {
```

Comment:
bikeshed: should this be called `transaction`?

Reply:
nvm, found the notes about not being a transaction down below

Reply:
Yeah, it is not a transaction.